yarn.lock

```diff
@@ -1086,6 +1086,51 @@
   resolved "https://registry.yarnpkg.com/@istanbuljs/schema/-/schema-0.1.2.tgz#26520bf09abe4a5644cd5414e37125a8954241dd"
   integrity sha512-tsAQNx32a8CoFhjhijUIhI4kccIAgmGhy8LZMZgGfmXcpMbPRUqn5LWmgRttILi6yeGmBJd2xsPkFMs0PzgPCw==
 
+"@jest/console@^24.9.0":
```

Comment:
since the versions in `package.json`s in this repo is now 25, any dependencies that depend on something from jest will now pull in here. Hopefully it doesn't cause type errors